src/appleseed.studio/mainwindow/mainwindow.cpp

```diff
@@ -899,10 +900,6 @@ void MainWindow::set_rendering_widgets_enabled(const bool is_enabled, const Rend
             // Set/clear rendering region.
             render_tab->set_render_region_buttons_enabled(
                 is_enabled && is_project_open && rendering_mode != RenderingMode::FinalRendering);
-
```

Comment:
Not sure why you removed this?

Reply:
This line was here to prevent the user from selecting things or change materials **while it's rendering**

Reply:
Removal of this snippet wasn't intended, it made sense with the previous iteration of the PR.
I'll add it back in.

> This line was here to prevent the user from selecting things or change materials while it's rendering
This isn't why I removed it. Also, it wasn't intended behaviour.
This wasn't affect material dropping at all.